package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@geoman-io/leaflet-geoman-free": "^2.16.0",
```

Comment:
Upgrade fixes a bug where if you drew a line on top of the marker, it would open the popup instead and you couldn't draw the line